sympy/printing/latex.py

```diff
@@ -155,12 +155,18 @@ def __init__(self, settings=None):
             "dot": r" \cdot ",
             "times": r" \times "
         }
-
-        self._settings['mul_symbol_latex'] = \
-            mul_symbol_table[self._settings['mul_symbol']]
-
-        self._settings['mul_symbol_latex_numbers'] = \
-            mul_symbol_table[self._settings['mul_symbol'] or 'dot']
+        try:
+            self._settings['mul_symbol_latex'] = \
+                mul_symbol_table[self._settings['mul_symbol']]
+        except KeyError:
+            self._settings['mul_symbol_latex'] = \
+                self._settings['mul_symbol']
+        try:
+            self._settings['mul_symbol_latex_numbers'] = \
+                mul_symbol_table[self._settings['mul_symbol'] or 'dot']
+        except KeyError:
+            self._settings['mul_symbol_latex_numbers'] = \
+                self._settings['mul_symbol'] or mul_symbol_table['dot']
```

Comment:
` or mul_symbol_table['dot']` was intended for `self._settings['mul_symbol']` being None. However, if `self._settings['mul_symbol']` is consisted of spaces(whitespace or LaTeX), it will not work.  I think there should be a check for it or another settings for `mul_symbol_latex_numbers`.